src/index.html

```diff
@@ -11,12 +11,64 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item"><a class="nav__link">gadgets</a></li>
+          <li class="nav__item"><a class="nav__link">tablets</a></li>
+          <li class="nav__item">
+            <a class="nav__link">photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">video</a>
```

Comment:
The `<a>` tag should have an `href` attribute to ensure it is a valid hyperlink. Consider adding a placeholder link like `href="#"` if the link is not yet defined.